Dnn.AdminExperience/ClientSide/Seo.Web/src/components/testUrl/index.jsx

```diff
@@ -86,7 +86,7 @@ class TestUrlPanelBody extends Component {
             isMultiLanguage: false,
             excludeAdminTabs: false,
             disabledNotSelectable: false,
-            roles: "0",
+            roles: props.adminRoleId ? props.adminRoleId : "0",
```

Comment:
Does it matter that `adminRoleId` is number and the fallback is a string?

Reply:
Good catch. It doesn't matter but it's not good practice. I'll correct. And I won't rant about having to work in classic JS instead of my preferred TS.